2.0/x-oca-event.md

```diff
@@ -2,47 +2,55 @@
+| domain_ref               | `object-ref`                | references a domain related to this event. must be of type `domain-name`                            |
+| registry_ref             | `object-ref`                | references windows registry data related to this event. must be of type `windows-registry-key`      |
+| network_ref              | `object-ref`                | references network traffic related to this event. must be of type `network-traffic`                 |
+| ip_refs                  | `list` of type `object-ref` | references any ip addresses related to this event. must be of type `ipv4-address` or `ipv6-address` |
+| user_ref                 | `object-ref`                | references the user related to this event. must be of type `user-account`                           |
+| severity                 | `number`                    | Severity of the event on a scale of 1-100.                                                          | 
+| start                    | `timestamp`                 | The start time in UTC of the event.                                                                 |
+| end                      | `timestamp`                 | The end time in UTC of the event.                                                                   |
+| timezone                 | `string`                    | The timezone where the event was discovered.                                                        |
+| duration                 | `string`                    | The duration of the event.                                                                          |
+| dataset                  | `string`                    | The dataset of the event.                                                                           |
+
+### Event Specific properties
+Certain events are important for security context which have their unique properties. The `x-oca-event` supports the following optional properties that should be populated only in certain `action` types.
+
+| property name | type     | description                                                                                                                                                                                                                           |
+|---------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| pipe_name     | `string` | In windows, there are many security rules that look for the pipe name property in a named pipe event. the value of this property should include the full pipe name in case of a named pipe event. see [example](#a-named-pipe-event). |
 
 ## Examples
 
-### an event indicating a network communication from an endpoint
+###An event indicating a network communication from an endpoint
```

Comment:
You need a space between ### and the first letter.

Reply:
+1